### PR TITLE
additional attributes only allow boolean

### PR DESCRIPTION
**Pull request by rebeccamadsen**
Only allows boolean additional attributes. Updates migration to remove any non-boolean additional attributes.

### CODE REVIEW

File: migrations/migrations/4.js [hunk 1]
```diff
@@ -78,6 +105,7 @@ const notes = `
 - Automatically rename **variable names** and **ordinal/categorical values** to meet stricter requirements. Only letters, numbers, and the symbols \`.\`, \`_\`, \`-\`, \`:\` will be permitted. Spaces will be replaced with underscore characters (\`_\`), and any other symbols will be removed. Variables that meet these requirements already **will not be modified**.
 - Add a numerical suffix (\`variable1\`, \`variable2\`, etc.) to any variables or categorical/ordinal values that clash as a result of these changes.
 - Rename node and edge types to ensure they are unique, and conform to the same requirements as variable names. Names that clash will get a numerical suffix, as above.
+- Removes any non-boolean additional attributes from prompts.
```
Comment (jthrilly) [inline migrations/migrations/4.js]: Can we put this point below the one after it, and replace the text with the following:

"Removes any non-boolean 'additional variables' from prompts. It was neccesary to simplify this feature, and so only boolean variable types will be supported moving forwards. Any non-boolean variables you created that will be removed by this migration will remain in your codebook, but will be marked 'unused'. You should review and remove this manually, or replace them with equivalent boolean variables.

**Review by jthrilly — CHANGES_REQUESTED** [review]
Looks good to me! Just the one small change to the wording and I think its good to merge.

File: migrations/migrations/4.js (hunk 1 above)
Comment (rebeccamadsen) [reply migrations/migrations/4.js]: Sounds good!

File: migrations/migrations/4.js [hunk 2]
```diff
@@ -105,8 +105,8 @@ const notes = `
...
-- Removes any non-boolean additional attributes from prompts.
-- **NOTE:** If you are using external network data, you must ensure that you update your column headings manually to meet the same requirements regarding variable names outlined above. See our revised [documentation on variable naming](https://documentation.networkcanvas.com/docs/key-concepts/variable-naming/).`;
+- **NOTE:** If you are using external network data, you must ensure that you update your column headings manually to meet the same requirements regarding variable names outlined above. See our revised [documentation on variable naming](https://documentation.networkcanvas.com/docs/key-concepts/variable-naming/).
+- Remove any non-boolean 'additional variables' from prompts. It was neccesary to simplify this feature, and so only boolean variable types will be supported moving forwards. Any non-boolean variables you created that will be removed by this migration will remain in your codebook, but will be marked 'unused'. You should review and remove this manually, or replace them with equivalent boolean variables.`;
```
Comment (jthrilly) [inline migrations/migrations/4.js]: necessary*

**Review by jthrilly — APPROVED** [review]
I fixed a typo, so please update the ref in the NC PR.

File: migrations/migrations/4.js (hunk 2 above)
Comment (rebeccamadsen) [reply migrations/migrations/4.js]: Ah good catch!

**Pull request merged**